app/controllers/concerns/sessions_management.rb

```diff
@@ -4,12 +4,12 @@ def new
   end
 
   def create
```

Comment:
Je ne comprends pas pourquoi on a 2 routes pour gérer les magic tokens. Cela devrait être géré dans un unique controller non ? Pour moi il ne devrait y avoir qu'un controller (le magic link) et celui-ci devrait être supprimé.

Reply:
A noter que les magic links ont 2 usages:

1/ Se connecter (User: log in, route A, Contact: montre les tokens dispos, route B)
2/ Transférer les jetons (L'user crée le magic link depuis le dashboard, le contact se connecte via route B)

Il doit y avoir au minima deux routes du coup pour gérer les magic token imho? A et B

Reply:
Oui en effet, c'était pourtant clair en voyant les noms des fichiers. My bad